### PR TITLE
Add wget to mock out downloads for importers

### PR DESCRIPTION
**Pull request by gregtatum**
This doesn't cover all of the importers, but it at least gets a mocked wget in place. This is in prep for #143.

Along with this I migrated one of the existing tests to use `run_task`

### CODE REVIEW

File: tests/test_data_importer.py [hunk 1]
```diff
@@ -47,36 +50,44 @@ def data_dir():
...
-)
+augmentation_params = [
+    ("sacrebleu_aug-upper_wmt19", is_upper_case, AUG_MIN_RATE, AUG_MAX_RATE),
+    ("sacrebleu_aug-upper-strict_wmt19", is_upper_case, 1.0, 1.0),
+    ("sacrebleu_aug-title_wmt19", is_title_case, AUG_MIN_RATE, AUG_MAX_RATE),
+    ("sacrebleu_aug-title-strict_wmt19", is_title_case, 1.0, 1.0),
+]
+
+
+@pytest.mark.parametrize("params", augmentation_params, ids=[d[0] for d in augmentation_params])
```
Comment (gregtatum) [inline tests/test_data_importer.py:90]: This is a drive-by, but I'm trying to make the keys for tests readable. Previously they were all just `params0` and `params1`, etc.

Now they are:

```
tests/test_data_importer.py::test_basic_corpus_import[mtdata-Neulab-tedtalks_test-1-eng-rus] PASSED                                                                                 [ 11%]
tests/test_data_importer.py::test_basic_corpus_import[opus-ELRC-3075-wikipedia_health_v1] PASSED                                                                                    [ 22%]
tests/test_data_importer.py::test_basic_corpus_import[flores-dev] PASSED                                                                                                            [ 33%]
tests/test_data_importer.py::test_basic_corpus_import[sacrebleu-wmt19] PASSED                                                                                                       [ 44%]
tests/test_data_importer.py::test_specific_augmentation[sacrebleu_aug-upper_wmt19] PASSED                                                                                           [ 55%]
tests/test_data_importer.py::test_specific_augmentation[sacrebleu_aug-upper-strict_wmt19] PASSED                                                                                    [ 66%]
tests/test_data_importer.py::test_specific_augmentation[sacrebleu_aug-title_wmt19] PASSED                                                                                           [ 77%]
tests/test_data_importer.py::test_specific_augmentation[sacrebleu_aug-title-strict_wmt19] PASSED                                                                                    [ 88%]
tests/test_data_importer.py::test_augmentation_mix PASSED                                                                                                                           [100%]
```

**Review by eu9ene — APPROVED** [review]
This is great and will make the tests run faster and be more reliable. One downside I see here is that we stop testing the real downloading from web resources. We run limited downloading in the pipeline CI though. I'm thinking we could add some integration tests for each importer later to cover issues with downloading and changes of the remote resources (say a dataset was re uploaded or an address has changed). For such tests we can skip testing advanced scenarios like for unit tests to make them fast enough.

**Comment by gregtatum** [comment]
So it sounds like the direction to go, is in pytest external resources should be mocked, and in the fast en-ru pipeline we should make sure and have real data with good coverage.
